generator/integration_tests/tests/golden_kitchen_sink_metadata_decorator_test.cc

```diff
@@ -466,6 +467,28 @@ TEST_F(MetadataDecoratorTest, ExplicitRoutingNestedField) {
   (void)stub.ExplicitRouting2(context2, request);
 }
 
+TEST_F(MetadataDecoratorTest, FixedMetadata) {
+  // We do this for a single RPC, we are using some knowledge of the
+  // implementation to assert that this is enough.
```

Comment:
"We use knowledge of the implementation to assert that testing a single RPC is sufficient."

Reply:
Fixed here and elsewhere.